integration-tests/features/steps/common.py

```diff
@@ -156,6 +156,13 @@ def delete_job(context, id):
     context.response = requests.delete(endpoint)
 
 
+@when("I change status for job with id {id} to {status}")
```

Comment:
What do you think about changing it to "set status" instead of "change status"? There is no previous state tracked. Otherwise LGTM.